clean_project/test_data/scriptdb.sql

```diff
@@ -0,0 +1,13 @@
+-- Create a database with a specific owner
+CREATE DATABASE index_db OWNER postgres;
+-- Create user with a specific option
+CREATE USER gianluca [ [ WITH ] option [ ... ] ]
+-- Grant all privileges on a specific database to given user
+grant all privileges on database index_db to gianluca;
+-- Grant SELECT, INSERT, UPDATE, DELETE on all tables to given user
+GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA public to gianluca;
+-- Grant all default privileges on a specific schema about tables and sequences to given user
+ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT ALL PRIVILEGES ON TABLES TO gianluca;
+ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT ALL PRIVILEGES ON sequences TO gianluca;
+-- Create a table with a specific fields
+CREATE TABLE Index(Word INTEGER, File text, Occurrence integer[]);
```

Comment:
the table name and column names should be all lowercase. also, since it's an array, it should be `occurrences`